### PR TITLE
Prepare v023

### PR DESCRIPTION
**Pull request by jvanhoefer**

### CODE REVIEW

File: doc/releasenotes.rst [hunk 1]
```diff
@@ -5,6 +5,37 @@ Release notes
 0.2 series
 ..........
 
+0.2.2 (2021-01-18)
+------------------
+
+* New Optimizers:
+  * FIDES (#506, #503 # 500)
+  * NLopt (#493)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:13]: pswarm is older, right? Then good.

File: doc/releasenotes.rst [hunk 2]
```diff
@@ -5,6 +5,37 @@ Release notes
...
+* Extended PEtab support:
+  * PySB import (#437)
+  * Support of PEtabs initializationPriors (#535)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:16]:
```suggestion
  * Support of PEtab's initializationPriors (#535)
```

File: doc/releasenotes.rst [hunk 3]
```diff
@@ -5,6 +5,37 @@ Release notes
...
+  * Support of priors parameterScale{Normal,Laplace}  (#520)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:17]:
```suggestion
  * Support of prior parameterScale{Normal,Laplace}  (#520)
```

File: doc/releasenotes.rst [hunk 4]
```diff
@@ -5,6 +5,37 @@ Release notes
...
+  * Example notebook  for synthetic data generation (#482)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:18]:
```suggestion
  * Example notebook for synthetic data generation (#482)
```

File: doc/releasenotes.rst [hunk 5]
```diff
@@ -5,6 +5,37 @@ Release notes
...
+* General new and improved functionality:
+  * Predictions (#544)
+  * Move tests to GitHub Actions (#524)
+  * Parallelize profile calculation (#532)
+  * Save x_guesses in `pypesto.problem (#494)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:23]:
```suggestion
  * Save `x_guesses` in `pypesto.problem` (#494)
```

**Review by yannikschaelte — APPROVED** [review]
Wow, that's a lot indeed.

File: doc/releasenotes.rst (hunk 1 above)
Comment (jvanhoefer) [reply doc/releasenotes.rst:13]: yep, already was there for the COMBINE

**Pull request merged**